src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -60,7 +67,13 @@ def __init__(self, name, button, panel, main_frame, main_data):
           Handles FastEMOverviewTab and FastEMAcquisitionTab.
 
         """
-        tab_data = guimod.FastEMMainTabGUIData(main_data)
+        tab_data = FastEMMainTabGUIData(main_data)
+        tab_data.tool.choices = {
+            TOOL_NONE,
+            TOOL_ACT_ZOOM_FIT,
+            TOOL_RULER,
+            TOOL_LINE,
```

Comment:
I tried out your code and the line tool does not work. Do you need the line tool?

Reply:
The line tool is to move the scale and extend the edges of the scale. Can you give it a try?

Reply:
Do you mean the move the edges of the ruler? That indeed works. I do sometimes get the following error:
```
09:09:11 ERROR   log            : Subscriber <odemis.util.weak.WeakMethodBound object at 0x7fb0207de700> raised exception when receiving value 4
Traceback (most recent call last):
  File "/home/pals/development/odemis/src/odemis/model/_vattributes.py", line 96, in notify
    l(v)
  File "/home/pals/development/odemis/src/odemis/util/weak.py", line 36, in __call__
    return self.f(ins, *arg, **kwargs)
  File "/home/pals/development/odemis/src/odemis/gui/cont/tabs/fastem_main_tab.py", line 181, in _on_tool_va
    self.vp.canvas.disable_drag()
  File "/home/pals/development/odemis/src/odemis/gui/comp/canvas.py", line 1447, in disable_drag
    self.abilities.remove(CAN_DRAG)
KeyError: 1
```

Reply:
My bad, yes to move the edges of the ruler. Nice catch I will fix this.

Reply:
I fixed it!